lib/error_tracker.ex

```diff
@@ -213,4 +213,35 @@ defmodule ErrorTracker do
 
     {error, occurrence}
   end
+
+  @doc """
+  Search errors with the given filters.
+
+  Example:
+
+      ErrorTracker.search(%{status: :unresolved, reason: "undefined"})
+
+  Valid filters are:
+    - `:reason`
+    - `:status`
+    - `:source_line`
+    - `:source_function`
+  """
+  @spec search(Ecto.Queryable.t(), %{atom() => any()}) :: Ecto.Queryable.t()
+  def search(query \\ Error, filters) when is_map(filters) do
```

Comment:
I'm not sure if exposing this API is something we should do, as it is tied to functionality in the UI.

If tomorrow we decide to change how filters in the UI work, this may need change and breaking the public API.

If this has been moved here for the testing changes, we can do the same by testing the liveview itself (which is something we are missing but we should add in the future).

What do you think?

Reply:
I am not sure either, so better be conservative and keep it in the LiveView directly. We can always expose it later.

Ping me when #56 is merged and I can rewrite this PR.

Reply:
merged!